mlrun/runtimes/nuclio/function.py

```diff
@@ -1241,6 +1247,13 @@ def get_url(
 
         return self._resolve_invocation_url("", force_external_address)
 
+    @staticmethod
+    def _resolve_v3io_access_key():
+        # if nuclio version < 1.13.11 return None, otherwise return "$generate"
```

Comment:
I think this comment should explain why you do what you do instead of explaining the code which is pretty clear.
does returning none here mean we don't pass the access key at all?